Src/AutoFakeItEasy/Properties/AssemblyInfo.cs

```diff
@@ -33,8 +33,9 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("3.50.2.0")]
-[assembly: AssemblyFileVersion("3.50.2.0")]
+[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyInformationalVersion("1.0.0.0")]
```

Comment:
I've updated the README file (see [here](https://github.com/AutoFixture/AutoFixture/blob/zvirja/calculate-build-version-based-on-repo-status/README.md#build)) and added information about build. Please review and let me know if you see how to improve the message :)